spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/Issue1254Test.java

```diff
@@ -39,11 +31,8 @@ void testUnresolvableReference() {
     @DisabledOnJre(JRE.JAVA_8)
     void testUncalledPrivateMethod() {
         performAnalysis(CLASS_LIST);
-        BugInstanceMatcher bugTypeMatcher = new BugInstanceMatcherBuilder().bugType("UPM_UNCALLED_PRIVATE_METHOD")
-                .build();
-        assertThat(getBugCollection(), containsExactly(2, bugTypeMatcher));
-        BugCollection bugCollection = getBugCollection();
-        bugCollection.findBug(null, "UPM_UNCALLED_PRIVATE_METHOD", 25);
-        bugCollection.findBug(null, "UPM_UNCALLED_PRIVATE_METHOD", 36);
+        assertBugTypeCount("UPM_UNCALLED_PRIVATE_METHOD", 2);
+        assertBugInMethod("UPM_UNCALLED_PRIVATE_METHOD", "Issue1254", "uncalledOuterMethod");
+        assertBugInMethod("UPM_UNCALLED_PRIVATE_METHOD", "Issue1254$Inner", "uncalledInnerMethod");
```

Comment:
The assert for the bugs was missing. The relevant bugs has no line info, but these refer to the same bugs. See https://github.com/spotbugs/spotbugs/blob/master/spotbugsTestCases/src/java11/Issue1254.java.